src/Vivait/StringGeneratorBundle/Generator/SecureBytesGenerator.php

```diff
@@ -37,7 +28,7 @@ public function setLength($length)
      */
     public function generate()
     {
-        return $this->secureRandom->nextBytes($this->length);
+        return random_bytes($this->length);
```

Comment:
This only seems to be available in PHP 7 - perhaps https://github.com/symfony/polyfill-php70 could be used?